doc/source/how-to/setup.rst

```diff
@@ -0,0 +1,451 @@
+please refer to `official Python documentation on venv module
+<https://docs.python.org/3/library/venv.html>`_.
+
+Check
+~~~~~
+Before creating a new virtual environment, you need to check if you are already
+working with one. To do so, run the following command:
+
+.. tabs::
+
+    .. group-tab:: Windows CMD
+
+        .. code-block:: text
+
+            where.exe python
+
+    .. group-tab:: Windows PowerShell
+
+        .. code-block:: text
+
+            where.exe python
+
+    .. group-tab:: macOS
+
+        .. code-block:: text
+
+            which python
+
+    .. group-tab:: Linux/UNIX
+
+        .. code-block:: text
+
+            which python
+
+Previous command will return the path to the currently used Python environment
+in your system. 
+
+Make sure it points towards your default installation and not towards a virtual
+environment. If this is the case, then refer to :ref:`Deactivate` section to
+deactivate your virtual environment.
+
+Create
+~~~~~~
+To create a virtual environment named `<venv>`, execute the following command:
+
+.. tabs::
+
+    .. group-tab:: Windows CMD
+
+        .. code-block:: text
+
+            py -m venv <venv>
+
+    .. group-tab:: Windows PowerShell
+
+        .. code-block:: text
+
+            py -m venv <venv>
+
+    .. group-tab:: macOS
+
+        .. code-block:: text
+
+            python -m venv <venv>
+
+    .. group-tab:: Linux/UNIX
+
+        .. code-block:: text
+            
+            python -m venv <venv>
+
+Usually, virtual environments are named as ``venv`` or ``.venv``.
+
+Activate
+~~~~~~~~
+To activate a virtual environment, execute the following command:
+
+.. tabs::
+
+    .. group-tab:: Windows CMD
+
+        .. code-block:: text
+
+            <venv>\Scripts\activate.bat
+
+    .. group-tab:: Windows PowerShell
+
+        .. code-block:: text
+
+            <venv>\Scripts\Activate.ps1
+
+    .. group-tab:: macOS
+
+        .. code-block:: text
+
+            source <venv>/bin/activate
+
+    .. group-tab:: Linux/UNIX
+
+        .. code-block:: text
+
+            source <venv>/bin/activate
+
+Deactivate
+~~~~~~~~~~
+
+.. tabs:: 
+
+    .. group-tab:: Windows CMD
+
+        .. code-block:: text
+
+            deactivate
+
+    .. group-tab:: Windows PowerShell
+
+        .. code-block:: text
+
+            deactivate
+
+    .. group-tab:: macOS
+
+        .. code-block:: text
+
+            deactivate
+
+    .. group-tab:: Linux/UNIX
+
+        .. code-block:: text
+
+            deactivate
+
+
+Git
+---
+.. raw:: html
+    
+    <div align="center">
+      <img src="https://github.com/git/git-scm.com/raw/main/public/images/logo%402x.png">
+      <br><br>
+    </div>
+
+`Git <https://git-scm.com/>`_ is an open-source version control system (VCS). It
+is used to track changes and register new ones in software related projects. Git
+will register author and dates too, so accurate tracking of the evolution of the
+software can be made.
```

Comment:
```suggestion
`Git <https://git-scm.com/>`_ is an open-source version control system (VCS). It
is used to track changes and register new content in software-related projects. Git
will register the author and date of the changes so that accurate tracking of the
software's evolution is available.
```